### PR TITLE
[DOCS] use wix-ui-icons-common icon list in storybook

### PR DESCRIPTION
**Pull request by gendeld**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6017

File: package.json [hunk 1]
```diff
@@ -153,6 +153,7 @@
     "execa": "^4.0.3",
     "express": "^4.17.1",
     "eyes.it": "^2.0.53",
+    "fuse.js": "^6.4.1",
```
Comment (shlomitc) [inline package.json:156]: this should probably become a dependency in `wix-ui-icons-common`

File: package.json [hunk 2]
```diff
@@ -179,6 +180,7 @@
     "storybook-snapper": "^1.0.63",
     "sync-ts": "^1.0.67",
     "typescript": "~3.7.5",
+    "use-debounce": "^4.0.0",
```
Comment (shlomitc) [inline package.json:183]: same here

**Review by shlomitc — COMMENTED** [review]
small changes. besides that, perfect

**Pull request merged**